### ISSUE TITLE
Does not support ASP.NET Core 1.0

### ISSUE THREAD

**Issue by lucasmansbridge** [reactjs/React.NET#286]
```
public class Startup
{
    public void ConfigureServices(IServiceCollection services)
    {
         services.AddReact(); // <-- here
    }
}
```

System.TypeLoadException: Could not load type 'Microsoft.Extensions.DependencyInjection.ServiceCollectionExtensions' from assembly 'Microsoft.Extensions.DependencyInjection.Abstractions, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.":"Microsoft.Extensions.DependencyInjection.ServiceCollectionExtensions

**Comment by Daniel15** [comment]
Wow, somehow I missed the announcement that the RTM version is out. I'll get this updated soon.

**Comment by ShikiGami** [comment]
I just sent a pull request ( https://github.com/reactjs/React.NET/pull/287 ) for this

**Comment by Daniel15** [comment]
Fixed in #287, thanks @ShikiGami! You can install version  `React.AspNet 2.4.1-dev-20160628-1635` or above from [the development/build server](http://reactjs.net/getting-started/download.html#development-builds) to install the new build. I'll do a proper release soon :)

**Comment by Daniel15** [comment]
Pushed this as version 2.5, will update documentation later today.
